gnat2goto/ireps/ireps-remove_bounds.adb

```diff
@@ -3,16 +3,30 @@ function Remove_Bounds (L : Irep_List) return Irep_List
 is
```

Comment:
New file - tell the reader what it does. 
And in particular why removing the bounds is required (this was 2 days work figuring out how to overcome a symex issue and there's not a comment in sight).  Is this done at the calling location? Good - better here. 

I raised this same issue on #115 - I thought it had been addressed?

Reply:
Done.